src/fanout/emqx_sdv_fanout_inflight.erl

```diff
@@ -0,0 +1,52 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2025 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%--------------------------------------------------------------------
+
+-module(emqx_sdv_fanout_inflight).
+
+-export([
+    create_tables/0,
+    insert/2,
+    delete/1,
+    lookup/1,
+    is_exist/1
+]).
+
+-include("emqx_sdv.hrl").
+
+%% @doc Create the tables.
+create_tables() ->
+    ets:new(?INFLIGHT_TAB, [
+        public,
+        named_table,
+        ordered_set,
```

Comment:
There are only single point reads in this module, so perhaps set is a better choice?